forge/forge/components/web/test_search.py

```diff
@@ -1,19 +1,22 @@
 import json
 
 import pytest
-from forge.components.web.search import WebSearchComponent
-from forge.utils.exceptions import ConfigurationError
 from googleapiclient.errors import HttpError
 from pydantic import SecretStr
 
-from autogpt.agents.agent import Agent
+from forge.components.web.search import WebSearchConfiguration
+from forge.utils.exceptions import ConfigurationError
+
+from . import WebSearchComponent
 
 
 @pytest.fixture
-def web_search_component(agent: Agent):
-    agent.web_search.config.google_api_key = SecretStr("test")
-    agent.web_search.config.google_custom_search_engine_id = SecretStr("test")
-    return agent.web_search
+def web_search_component():
+    config = WebSearchConfiguration(
+        google_api_key=SecretStr("test"),
+        google_custom_search_engine_id=SecretStr("test"),
```

Comment:
in what scenario does this work?

Reply:
It does work now, it mocks api replies I think. Replaced to:
```py
component = WebSearchComponent() # <-- this loads from env if present
if component.config.google_api_key is None:
    component.config.google_api_key = SecretStr("test")
if component.config.google_custom_search_engine_id is None:
    component.config.google_custom_search_engine_id = SecretStr("test")
return component
```